### PR TITLE
Update documentation.

### PR DESCRIPTION
**Pull request by arostamianfar**
There are still TODOs, but we can iterate on it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,59 +2,127 @@
...
-based on [Apache Beam](https://beam.apache.org/). The Python implementation
-supports running pipelines using
+This is a tool for transforming and processing
+[VCF](https://samtools.github.io/hts-specs/VCFv4.3.pdf) files in a scalable
+manner based on [Apache Beam](https://beam.apache.org/) using 
 [Dataflow](https://cloud.google.com/dataflow/) on Google Cloud Platform.
 
-### Pre-requisites
+It can be used to directly load VCF files to
+[BigQuery](https://cloud.google.com/bigquery/) in a scalable manner supporting
```
Comment (mhsaul) [inline README.md]: Remove "in a scalable manner" as its already mentioned above.

File: README.md [hunk 2]
```diff
@@ -2,59 +2,127 @@
...
+hunderds of thousands of files, millions of samples, and billions of records.
```
Comment (mhsaul) [inline README.md]: s/hunderds/hundreds

File: README.md [hunk 3]
```diff
@@ -2,59 +2,127 @@
...
+
+### Prerequisites
+
+1.  Setup a [Google Cloud account](https://cloud.google.com/) and
+    [create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects),
+    if you don't have one already.
```
Comment (mhsaul) [inline README.md]: s/if you don't have one already/

File: README.md [hunk 4]
```diff
@@ -2,59 +2,127 @@
...
+  imageName: gcr.io/gcp-variant-transforms/gcp-variant-transforms
+  cmd: |
+    ./opt/gcp_variant_transforms/bin/vcf_to_bq \
+      --project my-project \
+      --input_pattern gs://my-bucket/vcffiles/*.vcf \
+      --output_table my-project:my-bigquery-dataset.my-bigquery-table \
+      --staging_location gs://my-bucket/staging \
+      --temp_location gs://my-bucket/temp \
+      --job_name vcf-to-bigquery \
+      --runner DataflowRunner
```
Comment (mhsaul) [inline README.md]: Placing "<>" around parameters that need to be replaced can be helpful to users.

File: README.md [hunk 5]
```diff
@@ -2,59 +2,127 @@
...
+with your project name, `gs://my-bucket/temp/runner_logs` with a Cloud Storage
+folder to store the logs from the pipeline.
+
+```bash
+gcloud alpha genomics pipelines run \
+    --project my-project \
+    --pipeline-file vcf_to_bigquery.yaml \
+    --logging gs://my-bucket/temp/runner_logs \
+    --zones us-west1-b \
+    --service-account-scopes https://www.googleapis.com/auth/bigquery
```
Comment (mhsaul) [inline README.md]: Placing "<>" around parameters that need to be replaced can be helpful to users.

File: README.md [hunk 6]
```diff
@@ -68,72 +136,23 @@ Example command for DataflowRunner:
...
-  --project projectname \
-  --staging_location gs://bucket/staging \
-  --temp_location gs://bucket/temp \
-  --job_name vcf-to-bq \
+  --input_pattern gs://my-bucket/vcffiles/*.vcf \
+  --output_table my-project:my-bigquery-dataset.my-bigquery-table \
+  --project my-project \
+  --staging_location gs://my-bucket/staging \
+  --temp_location gs://my-bucket/temp \
+  --job_name vcf-to-bigquery \
```
Comment (mhsaul) [inline README.md]: Placing "<>" around parameters that need to be replaced can be helpful to users.

File: README.md [hunk 7]
```diff
@@ -68,72 +136,23 @@ Example command for DataflowRunner:
...
 ```bash
 python -m gcp_variant_transforms.vcf_to_bq \
-  --input_pattern gs://bucket/vcfs/vcffile.vcf \
-  --output_table projectname:bigquerydataset.tablename  \
-  --project projectname \
-  --staging_location gs://bucket/staging \
-  --temp_location gs://bucket/temp \
-  --job_name vcf-to-bq \
+  --input_pattern gs://my-bucket/vcffiles/*.vcf \
+  --output_table my-project:my-bigquery-dataset.my-bigquery-table \
```
Comment (mhsaul) [inline README.md]: I think the dataset and table actually can't have "-" in them. Might be good to change them to "_" even though they're just dummy values.

File: README.md (hunk 1 above)
Comment (arostamianfar) [reply README.md]: Done. Good idea!

File: README.md (hunk 2 above)
Comment (arostamianfar) [reply README.md]: Nice catch!

File: README.md (hunk 3 above)
Comment (arostamianfar) [reply README.md]: Done.

File: README.md (hunk 4 above)
Comment (arostamianfar) [reply README.md]: I actually disagree. If you recall, we recently had a case where a user accidentally left the closing bracket in place. I think this is more natural (albeit, not ideal).

File: README.md (hunk 5 above)
Comment (arostamianfar) [reply README.md]: See above.

File: README.md (hunk 7 above)
Comment (arostamianfar) [reply README.md]: Good point! Done.

File: README.md (hunk 6 above)
Comment (arostamianfar) [reply README.md]: See above.

**Pull request merged**